packages/hash/README.md

```diff
@@ -26,7 +26,7 @@ To run HASH locally, please follow these steps:
     ## ≥ 2.17
     
     node --version
-    ## ≥ 16.13 and < 17.00 (`@opensearch-project/opensearch` currently does not support node 17)
+    ## ≥ 16.13, < 17 or ≥ 18.0, < 19
```

Comment:
The original impetus for investigating this was to make it less the version requirement confusing to new users – in that spirit:
a) what is the issue with using v17?
b) do we need to mention 19? it's not available for months

Reply:
Node 17 goes end of life in less than a month, so I did not include it.
https://nodejs.org/en/about/releases/

![image](https://user-images.githubusercontent.com/608862/166652715-6dfae8e6-8a38-49ca-bd90-3b0b64ebd70c.png)

Odd version numbers are less supported by library authors, so it’s generally not recommended to use them. The goal of these versions is to prepare for the next even version, which becomes LTS at some point. For example, odd versions may lack binaries for some packages so they need to be built locally. That’s a nightmare on some machines.

Having said that, happy to go for: `## ≥ 16.13` to make the docs looks a bit simpler. Formally speaking, this will be an overpromise because we don’t really know if Node 17 is supported or not. Even if we test things with it now, everything might break with a new dependency. Running CI for Node 17 is technically possible, but does not feel worth it.

We can also do `## ^16.13 or ^18.0`. I’m not sure if `^` is as well-understood as `≥ <` amongst newstarters.

Reply:
We could say something like this?

> Supported and tested on the Latest and LTS versions of Node:
>  * Latest: Node 18.x >= 18.0
>  * Active LTS: Node 16.x >= 16.13

Reply:
Let's just say `## ≥ 16.13` and if someone reports it not working on 17 we can deal with it then – I don't think the additional burden in deciphering anything complex is worth it, until we know there's an actual issue.

Reply:
We could add the detail somewhere else down the README, but we want the 'getting set up' instructions to be as painless as possible.

Reply:
Yeah we can add that to the Getting started section too. I guess the question is what to write here to keep things aligned:

```
git --version
## ≥ 2.17

node --version
## ≥ 16.13, < 17 or ≥ 18.0, < 19

yarn --version
## ≥ 1.16

docker --version
## ≥ 20.10
```

Reply:
`≥ 16.13`

Reply:
Done! Saw your comments only after finishing mine 😅